libs/solid/skeleton/src/lib/components/base-layout/base-layout.component.html

```diff
@@ -20,7 +20,7 @@
         </button>
         <div class="bottom"></div>
       </div>
-      <div>
+      <div id="step4">
         <button (click)="Glossary?.toggle()" mat-icon-button>
           <mat-icon aria-label="Menü öffnen" svgIcon="glossary"></mat-icon>
```

Comment:
Das `aria-label` müsste hier "Glossar öffnen" heißen, sehe ich gerade. Einfach beim nächsten commit mit fixen.

Reply:
Ich fixe das mal eben in einem anderen PR (for technical reasons)

Reply:
Und schon gibt es einen Merge Conflict 😅 
Aber das ist leicht gelöst.